cli/pkg/kctrl/cmd/package/installed/create_or_update.go

```diff
@@ -310,6 +312,20 @@ func (o CreateOrUpdateOptions) update(client kubernetes.Interface, kcClient kccl
 		return nil
 	}
 
+	// Pause reconciliation so that kctrl can tail status if an existing values secret is updated
+	reconciliationPaused := false
```

Comment:
This variable is not entirely necessary, we can chuck it out
I thought it improves readability but happy to remove it